testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -121,19 +120,23 @@ class MiniAppDisplayActivity : BaseActivity() {
                         callback
                     )
                 }
+
+                override fun share(content: String) {
+                    val sendIntent: Intent = Intent().apply {
+                        action = Intent.ACTION_SEND
+                        putExtra(Intent.EXTRA_TEXT, content)
+                        type = "text/plain"
+                    }
+                    startActivity(sendIntent)
```

Comment:
if you use `startActivity(Intent.createChooser(sendIntent, null))` then, you can adopt some extra features provided by `Intent`.